website/docs/tutorial/1b-setting-up-snowflake.md

```diff
@@ -3,14 +3,299 @@ title: Setting up with Snowflake
 id: setting-up-snowflake
 ---
 
-This guide is under construction
-
 ## Get started with Snowflake
 
+1. To get started, sign up for a free trial by following [this link](https://signup.snowflake.com/). 
+
+2. Then fill out the signup form.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_free_trial_signup_form.png" title="Snowflake Free Trial Signup Form" />
+</p>
+
+3. Select the Enterprise edition, choose a cloud provider and region, and agree to the terms of service. There are some organizational questions to think about when choosing a cloud provider for a full implementation, which you can read more about [here](https://docs.snowflake.com/en/user-guide/intro-cloud-platforms.html). For the purposes of this setup, all cloud providers and regions will work so choose whichever you’d like. Finally, click "GET STARTED".
+
+<p align="center">
+<Lightbox  src="/img/snowflake_tutorial/snowflake_account_configuration.png" title="Snowflake Account Configuration" />
+</p>
+
+4. After submitting the signup form you should receive an email asking you to activate your account. Click on the link in the email and a new tab will open up where you’ll create your username and password. Enter in all that info and click "Get started".
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_account_signup.png" title="Snowflake Account Signup" />
+</p>
+
+5. Congrats! Your workspace is ready for some data. Feel free to check out any of the getting started tooltips that Snowflake provides in the UI to familiarize yourself before moving on to the next section.
+
+<p align="center">
+<Lightbox src="/img/snowflake_tutorial/snowflake_workspace.png" title="Snowflake Workspace" />
+</p>
+
+
 ## Load sample data into Snowflake
 
-## Set up a project to connect dbt Cloud to Snowflake (partner connect)
+1. Now we’re ready for some sample data. The data used here is stored as CSV files in a public S3 bucket and the following steps will guide you through how to prepare your Snowflake account for that data and upload it.
```

Comment:
```suggestion
Now we’re ready for some sample data. The data used here is stored as CSV files in a public S3 bucket and the following steps will guide you through how to prepare your Snowflake account for that data and upload it.
```
suggestion: since this is setting context rather than an action taken by the learner, I'd suggest making this a normal paragraph item.

Reply:
Sounds good, easy fix